content/quizzes/en.yaml

```diff
@@ -2593,10 +2593,10 @@ levels:
             code: |-
                 courses = appetizer, main course, dessert
                 names = Timon, Onno
-                {for} name {in} names
-                	{for} course {in} courses
-                        	food = {ask} name ', what would you like to eat as your ' course '?'
-                        	{print} name ' orders ' food ' as their ' course
+                {for} name in names
```

Comment:
Ah I think here merge issues have caused some good changes to also be gone, I will take a further look!